pahole/patches/libff-to-latest.patch

```diff
@@ -0,0 +1,91 @@
+--- btf_encoder.c
```

Comment:
same question about this patch file

Reply:
This patch comes from the Alpine package here https://git.alpinelinux.org/aports/tree/community/pahole/0001-libbpf-Update-libbpf-to-the-latest-git-HEAD.patch